package.json

```diff
@@ -161,9 +158,7 @@
   "cordova": {
     "plugins": {
       "cordova-plugin-whitelist": {},
```

Comment:
Wonder if we need this line if ```cordova-plugin-whitelist``` has been removed above?
Same for ```cordova-plugin-ionic-webview``` and ```cordova-plugin-ionic-keyboard```.

Can we also check if they can then be removed from ```config.xml```?

Reply:
good spot, they done seem to be relevent anymore.  will remove and test they still work correctly.